Makefile

```diff
@@ -7,7 +7,7 @@ all: venv
 clean:
 	rm -rf dist venv .tox *.egg-info *.log*
 
-.PHONY: venv
+.PHONY: all clean venv
```

Comment:
nitpick: if we're going to have one .PHONY, can it be at the top of the file?

alternatively, I think you can attach a .PHONY to each target

```makefile
.PHONY: clean
clean:
    ...

.PHONY: venv
venv:
    ...
```